### DISCUSSION TITLE
Drop database

### DISCUSSION THREAD

**Discussion by Peterculazh** [prisma/prisma#17904] [Q&A]
Hello, I'm wondering is there a way to drop database with CLI?
I know about dropping by raw query, but I don't want to use it because of hardcoding name of database in code and query, that's taking away the flexibility with environment for testing

**Comment by ludralph** [comment]
Hi @Peterculazh 👋 

[DB execute](https://www.prisma.io/docs/reference/api-reference/command-reference#db-execute) command applies an sql script to the database. You can also learn about it's usage in this [article](https://www.prisma.io/blog/prisma-migrate-dx-primitives#prisma-db-execute).

**Reply by Peterculazh** [reply]
Thanks, tried to figure out the any ways to drop DB by `prisma db execute`, but hadn't any success because it's still not available to pass variables to sql script from environment file
Moreover Prisma is connecting directly to DB so on drop it'll throw error that there are connection so I can't to drop DB

Looks like Prisma have no such base functionality at all, which is not understandable for me
Seems right now is just easier to use another pg native library and just connect to Postgres server and drop database and disconnect

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
